styles.css

```diff
@@ -0,0 +1,28 @@
+body {
+    background-color: rgb(239, 234, 234);
+    color: black;
+    text-align: center;
+    font-family: 'Montserrat', sans-serif;
+    margin: 50px 100px 50px 100px;
+  }
+  #title{
+    text-align: center;
+  }
+  #img-div{
+    text-align: center;
+    display: block;
+    background-color: white;
+    width: 100%;
+  }
+
+  #image{
+    max-width: 1000px;
+    height: auto;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  #tribute-info{
+    font-size: medium;
+    text-align: center;
+  }
```

Comment:
Kindly pay attention to the linters errors, I think your styles.css needs to be corrected so every style you included in your document will appear.